packages/flutter_tools/lib/src/vmservice.dart

```diff
@@ -1327,6 +1331,7 @@ class FlutterView extends ServiceObject {
     final String viewId = id;
     // When this completer completes the isolate is running.
     final Completer<Null> completer = new Completer<Null>();
+    await owner.vm.vmService.subscribedToIsolateEvents();
```

Comment:
Subscription is supposed to happen at line 1336 below, right?

Reply:
Yes, but we don't await it there.

Reply:
I find https://github.com/flutter/flutter/pull/16697 much simpler :-)